Rn okta364354 remove dev console (#1905)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/protect-your-api/configure-packages/index.md

#### Things you need

> * **Okta Domain** &mdash; The Okta Domain can be found on the Admin Console's global header in the upper-right corner. Click the section which displays your email and company name.  A drop-down menu will appear and display general org information including the full Okta domain (e.g. subdomain.okta.com).

Reviewer: ```suggestion
* **Okta Domain** &mdash; The Okta Domain can be found on the Admin Console's global header in the upper-right corner of the page. Click the section that displays your email and company name.  A drop-down menu appears and displays general org information including the full Okta domain (for example, `subdomain.okta.com`).
```
